apps/condo/migrations/20230511104849-0269_remove_banktransaction_bank_transaction_unique_number_date_organization_and_more.js

```diff
@@ -0,0 +1,42 @@
+// auto generated by kmigrator
+// KMIGRATOR:0269_remove_banktransaction_bank_transaction_unique_number_date_organization_and_more:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMSBvbiAyMDIzLTA1LTExIDA1OjQ4Cgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucywgbW9kZWxzCgoKY2xhc3MgTWlncmF0aW9uKG1pZ3JhdGlvbnMuTWlncmF0aW9uKToKCiAgICBkZXBlbmRlbmNpZXMgPSBbCiAgICAgICAgKCdfZGphbmdvX3NjaGVtYScsICcwMjY4X3JlbmFtZV9zZXR1cHVybF9hY3F1aXJpbmdpbnRlZ3JhdGlvbl9hcHB1cmxfYW5kX21vcmUnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuUmVtb3ZlQ29uc3RyYWludCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nYmFua3RyYW5zYWN0aW9uJywKICAgICAgICAgICAgbmFtZT0nQmFua190cmFuc2FjdGlvbl91bmlxdWVfbnVtYmVyX2RhdGVfb3JnYW5pemF0aW9uJywKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuUmVtb3ZlQ29uc3RyYWludCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nYmFua3RyYW5zYWN0aW9uJywKICAgICAgICAgICAgbmFtZT0nQmFua190cmFuc2FjdGlvbl91bmlxdWVfaW1wb3J0SWRfaW1wb3J0UmVtb3RlU3lzdGVtX29yZ2FuaXphdGlvbicsCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFkZENvbnN0cmFpbnQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2Jhbmt0cmFuc2FjdGlvbicsCiAgICAgICAgICAgIGNvbnN0cmFpbnQ9bW9kZWxzLlVuaXF1ZUNvbnN0cmFpbnQoY29uZGl0aW9uPW1vZGVscy5RKCgnZGVsZXRlZEF0X19pc251bGwnLCBUcnVlKSksIGZpZWxkcz0oJ29yZ2FuaXphdGlvbicsICdpbXBvcnRSZW1vdGVTeXN0ZW0nLCAnaW1wb3J0SWQnKSwgbmFtZT0nQmFua190cmFuc2FjdGlvbl91bmlxdWVfb3JnYW5pemF0aW9uX2ltcG9ydFJlbW90ZVN5c3RlbV9pbXBvcnRJZCcpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Remove constraint Bank_transaction_unique_number_date_organization from model banktransaction
+--
+DROP INDEX IF EXISTS "Bank_transaction_unique_number_date_organization";
+--
+-- Remove constraint Bank_transaction_unique_importId_importRemoteSystem_organization from model banktransaction
+--
+DROP INDEX IF EXISTS "Bank_transaction_unique_importId_importRemoteSystem_organization";
+--
+-- Create constraint Bank_transaction_unique_organization_importRemoteSystem_importId on model banktransaction
+--
+CREATE UNIQUE INDEX "Bank_transaction_unique_organization_importRemoteSystem_importId" ON "BankTransaction" ("organization", "importRemoteSystem", "importId") WHERE "deletedAt" IS NULL;
```

Comment:
Keep in mind that identifier "Bank_transaction_unique_organization_importRemoteSystem_importId" exceeds 63 characters. Postgres cuts identifier names up to maximum length of 63 characters.
You should do nothing with it. Just FYI.